libs/fixture-generators/src/cli/generate-election-package/index.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  ElectionPackage,
+  formatBallotHash,
+  formatElectionPackageHash,
+  safeParseElection,
+} from '@votingworks/types';
+import { readdirSync, readFileSync } from 'node:fs';
+import { assertDefined } from '@votingworks/basics';
+import { readElectionPackageFromFile } from '@votingworks/backend';
+import path from 'node:path';
+import { stdout } from 'node:process';
+import { generateElectionPackage } from '../../generate-election-package';
+
+// A script to generate an election package.
+// Usage: ./generate-election-package <election_fixtures_directory_path> <is_multi_language> <force_restranslate>
```

Comment:
API suggestions to improve clarity and better mesh with existing scripts in this lib:
- Take a path to the election.json rather than requiring that it is named `baseElection.json` so the script can be used more flexibly
- Instead of `force_retranslate`, optionally take a path to the election package that should be used as a translation cache (e.g. `--reuse-translations <path_to_package>`)
- Change `is_multi_language` to a flag `--multi-language`. It's a bit confusing just seeing a `1` in the command
- Require an output path to be specified `--outputPath <path>`

Take them or leave them, not blocking